cluster/gce/win1803/install-logging-agent.psm1

```diff
@@ -30,7 +32,18 @@ function InstallAndStart-LoggingAgent {
   Remove-Item `
       -Force `
       -ErrorAction Ignore `
-      'C:\Program Files (x86)\Stackdriver\LoggingAgent\Main\pos\winevtlog.pos\worker0\storage.json'
+      ("$STACKDRIVER_ROOT\LoggingAgent\Main\pos\winevtlog.pos\worker0\" +
+       "storage.json")
+
+  if (Test-Path $STACKDRIVER_ROOT) {
+    # TODO: check $REDO_STEPS variable here.
```

Comment:
Do you want to check `$REDO_STEPS` here before merging?

Reply:
I'll do this in a follow-up - I intend to factor a small "common" lib out of k8s-node-setup.psm1 that will contain this variable and other common code. That seemed like an invasive enough change to separate it from this PR.